apps/AppStandalone/README.md

```diff
@@ -5,39 +5,156 @@ A new Flutter project.
 ## Getting Started
 
 FlutterFlow projects are built to run on the Flutter _stable_ release.
+
+---
+
+This guide walks you through the steps required to run the app locally on your machine. Please
+follow the instructions carefully to ensure a smooth setup.
+
+### Prerequisites
+
+Before starting, make sure you have the following installed:
+
+- Flutter SDK
+- Dart SDK
+- Xcode (for iOS)
+- Android Studio (for Android)
+- CocoaPods (for iOS dependencies)
+- Firebase account
+
+### Setup Instructions
+
+1. **Get Flutter Dependencies**:
+    ```
+    flutter pub get
+    ```
+
+2. **Install iOS Pods**:
+   Navigate to the iOS directory and install the CocoaPods dependencies:
+    ```
+    cd ios
+    pod install
+    pod repo update
+    ```
+
+3. **Environment Configuration**:
+   Rename the environment configuration file:
+    ```
+    mv .env.template .env
+    ```
+
+4. **Firebase Project**:
+    - Create a new project in Firebase Console.
+    - Activate the FlutterFire CLI tool:
+        ```
+        dart pub global activate flutterfire_cli
+        ```
+    - Configure FlutterFire with your Firebase project:
+        ```
+        flutterfire configure
+        ```
+      Follow the prompts to select your project and target platforms (Android, iOS, Web).
+
+5. **API Keys**:
+   Add your Firebase and other necessary API keys to the `.env` file.
+
+6. **Run Build Runner**:
+   Generate necessary files with Build Runner:
+    ```
+    dart run build_runner build
+    ```
+
+7. **Configure Xcode**:
+    - Open your iOS project in Xcode:
+        ```
+        open ios/Runner/Runner.xcworkspace
```

Comment:
I believe this should just be [ios/Runner.xcworkspace](https://github.com/BasedHardware/Friend/tree/main/apps/AppStandalone/ios)
```suggestion
        open ios/Runner.xcworkspace
```